boards/sodaq-autonomo/Makefile.dep

```diff
@@ -0,0 +1,4 @@
+# Not sure if this is needed
```

Comment:
This comment can be removed as you added `gpio_params.h` where you configured a saul entry for the on-board LED. So leaving the `saul_gpio` can be useful.